src/components/McaLogo.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FunctionComponent } from "react";
+
+export const McaLogo: FunctionComponent = (): JSX.Element => (
+  <>
+    <a href="https://gov.uk/mca">
+      <img src="../assets/images/mca-logo.png" alt="mca logo" />
+    </a>
+    <br />
```

Comment:
I would create a styles file for this and put some padding on the element rather than having multiple break elements here